spec/user_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "user"
+require 'db_helpers'
+RSpec.describe User do
+  describe "New user" do
+    it "creates a new user" do
+      newuser = instance_double(User)
+      allow(newuser).to receive(:username).and_return("test")
+      allow(newuser).to receive(:email).and_return("test@test.com")
+      allow(newuser).to receive(:password).and_return("test")
+      expect(newuser.username).to eq "test"
+      expect(newuser.email).to eq "test@test.com"
+      expect(newuser.password).to eq "test"
+    end
+  end
+
+  describe '.create' do
+    it 'creates a new user' do
+      user = User.create(username: 'test_user', email: 'test@test.com', password: 'test_password')
```

Comment:
Should probably add a setup_test_database helper to truncate the database before each test

Reply:
will do this afternoon